client/src/App.tsx

```diff
@@ -63,6 +63,11 @@ const App = () => {
             .then(json => setEvents(json.items));
     }, [])
 
+    function logSometimes(...args:any[]){
+        if (((new Date()).getSeconds()%5) == 0){
```

Comment:
That's a lot of redundant parentheses :P (also `===` 😠) Most of those parentheses aren't needed

```suggestion
        if (new Date().getSeconds() % 5 === 0){
```

Though I suppose this is just for debugging